File: app/src/main/java/cn/intret/app/picgo/app/MyAppGlideModule.java
package cn.intret.app.picgo.app;

import android.content.Context;

import com.bumptech.glide.GlideBuilder;
import com.bumptech.glide.Registry;
import com.bumptech.glide.annotation.GlideModule;
import com.bumptech.glide.load.engine.cache.LruResourceCache;
import com.bumptech.glide.module.AppGlideModule;

/**
 * Customize the Glide
 */
@GlideModule
public final class MyAppGlideModule extends AppGlideModule {
    @Override
    public void applyOptions(Context context, GlideBuilder builder) {
//        builder.setMemoryCache(new LruResourceCache(20 * 1024 * 1024));

    }

    @Override
    public boolean isManifestParsingEnabled() {
        return false;
    }
}

File: app/src/main/java/cn/intret/app/picgo/app/MyAppGlideModule.kt
package cn.intret.app.picgo.app

import android.content.Context

import com.bumptech.glide.GlideBuilder
import com.bumptech.glide.Registry
import com.bumptech.glide.annotation.GlideModule
import com.bumptech.glide.load.engine.cache.LruResourceCache
import com.bumptech.glide.module.AppGlideModule

/**
 * Customize the Glide
 */
@GlideModule
class MyAppGlideModule : AppGlideModule() {
    override fun applyOptions(context: Context, builder: GlideBuilder) {
        //        builder.setMemoryCache(new LruResourceCache(20 * 1024 * 1024));
    }

    override fun isManifestParsingEnabled(): Boolean {
        return false
    }
}
